foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-StoragePolicySetDefinition.json

```diff
@@ -56,6 +56,795 @@
+                        },
+                        "parameters": {
+                            "effect": {
+                                "type": "String",
+                                "metadata": {
+                                    "displayName": "Effect",
+                                    "description": "Enable or disable the execution of the policy"
+                                },
+                                "allowedValues": [
+                                    "Deny",
+                                    "Audit",
+                                    "Disabled"
+                                ],
+                                "defaultValue": "Deny"
+                            },
+                            "allowedCopyScope": {
+                                "type": "String",
+                                "metadata": {
+                                    "displayName": "Allowed Copy Scope",
+                                    "description": "Specify the allowed copy scope."
+                                },
+                                "allowedValues": [
+                                    "AAD",
+                                    "PrivateLink"
+                                ],
+                                "defaultValue": "AAD"
+                            }
+                        },
+                        "policyRule": {
+                            "if": {
+                                "allOf": [
+                                    {
+                                        "field": "type",
+                                        "equals": "Microsoft.Storage/storageAccounts"
+                                    },
+                                    {
+                                        "anyOf": [
+                                            {
+                                                "field": "Microsoft.Storage/storageAccounts/allowedCopyScope",
+                                                "exists": "false"
+                                            },
+                                            {
+                                                "field": "Microsoft.Storage/storageAccounts/allowedCopyScope",
+                                                "notEquals": "[[parameters('allowedCopyScope')]"
+                                            }
+                                        ]
+                                    }
+                                ]
+                            },
+                            "then": {
+                                "effect": "[[parameters('effect')]"
+                            }
+                        }
+                    },
+                    "name": "Deny-Storage-CopyScope"
+                },
+                {
+                    "properties": {
+                        "displayName": "Encryption for storage services should be enforced for Storage Accounts",
```

Comment:
How will this policy work in conjunction with /providers/Microsoft.Authorization/policyDefinitions/b5ec538c-daa0-4006-8596-35468b9148e8 (encryption scopes should use CMK) which we are already including?